### ISSUE TITLE
random?

### ISSUE THREAD

**Issue by hmelberg** [newville/asteval#35]
I (foolishly?) tried:

`aeval('x = random.normal(0,1)')`

and got

```
NameError
x = random.normal(0,1)
     ^^^
name 'random' is not defined
```
Is is possible to draw random numbers from distributions using asteval?

Perhaps it would create a security problem to give access to the functions in the random module in numpy, or would it be OK? 

Maybe the random subfunctions could be added into the namespace at startup with the other numpy functions.

Willing to help with this if you think it is a good idea.

Great package, thanks!

(edited: language and typos)

**Comment by newville** [comment]
@hmelberg Sorry for the trouble.   This is fixed in the github master branch.
That suggests it really is time to release a new version!

**Comment by newville** [comment]
@hmelberg Oh, wait, I was wrong.  It looks like `random` was removed at some point... investigating.

**Comment by newville** [comment]
@hmelberg `random` was removed along with many other imported symbols from numpy in June, 2015.  I'll put `random` back.

**Comment by hmelberg** [comment]
Great! Thanks!
(I use your package to make a framework for calculating the expected value of different choice alternatives in a decision treee, and often the costs, benefits, and probabilities are drawn from distributions).

**Comment by newville** [comment]
@hmelberg  this is now fixed in 4cfdedf99bb41e5ba831aeb553e342d92ed954b0

**Comment by newville** [comment]
fixed in 0.9.9
